spec/app/domain/authentication/authn-azure/application_identity_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Authentication::AuthnAzure::ApplicationIdentity do
+  include_context "azure setup"
+
+  let(:user_assigned_identity_service_id_scoped_annotation) { double("UserAssignedIdentityAnnotation") }
+  let(:system_assigned_identity_service_id_scoped_annotation) { double("UserAssignedIdentityServiceIdScopeAnnotation") }
+
+  before(:each) do
+    define_host_annotation(host_annotation_type:  user_assigned_identity_service_id_scoped_annotation,
+                           host_annotation_key:   "#{granular_annotation_type}/user-assigned-identity",
+                           host_annotation_value: "some-user-assigned-service-id-scoped-value")
```

Comment:
Please change to:
```
define_host_annotation(
  host_annotation_type:  user_assigned_identity_service_id_scoped_annotation,
  host_annotation_key:   "#{granular_annotation_type}/user-assigned-identity",
   host_annotation_value: "some-user-assigned-service-id-scoped-value",
)
```

Do this everywhere

Reply:
THIS IS NOT RESOLVED!

Reply:
this was resolved..